docker/tools/Dockerfile

```diff
@@ -1,3 +1,50 @@
+# Build Kopia binary
+FROM golang:1.19-bullseye AS builder
+
+# Refers to https://github.com/kopia/kopia/commit/317cc36892707ab9bdc5f6e4dea567d1e638a070
+ARG kopiaBuildCommit=317cc36
```

Comment:
Should we use the latest commit by default? Maybe a master branch?

Reply:
Since we are cloning, it would be latest commit always so we can leave it empty as well. Setting master might confuse others if it should be branch instead of commit.

Reply:
sounds good